New billing FAQ documentation

Abishya opened the pull request:
This documentation covers frequently asked questions around subscription plans, payments, invoices and billing in general.

Abishya committed:
New billing FAQ documentation

This documentation covers frequently asked questions around subscription plans, payments, invoices and billing in general.

charislam reviewed (approved):
LGTM!

Just correct the misspellings that the CI check flagged (we use AmEng so 'z' instead of 's')

Abishya committed:
fix spelling

Corrected the misspellings ('z' instead of 's')

This documentation covers frequently asked questions around subscription plans, payments, invoices and billing in general

Abishya commented:
@charislam I have fixed the spelling. Could you please approve my PR?
